setup.py

```diff
@@ -4,20 +4,20 @@
 
 extras_require = {
     "test": [  # `test` GitHub Action jobs uses this
-        "pytest>=6.0",  # Core testing package
+        "pytest>=6.0,<8",  # Core testing package
         "pytest-xdist",  # multi-process runner
         "pytest-cov",  # Coverage analyzer plugin
-        "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
+        "hypothesis>=6.70.0,<7",  # Strategy-based fuzzer
     ],
     "lint": [
-        "black>=22.12.0",  # auto-formatter and linter
-        "mypy>=0.991",  # Static type analyzer
+        "black>=23.1.0,<24",  # auto-formatter and linter
+        "mypy>=1.1.1,<2",  # Static type analyzer
```

Comment:
Are we finally switching to support the newest version of mypy? I know core and most of our other plugins are still v0.991 and we ran into errors with 1.0 with the Starknet plugin before.

Reply:
I would like to upgrade mypy in core as well as a follow-up task.
But these are only lint deps, so they don't depend on core or each other. You can create separate dev envs for each while waiting for extra-dependencies to catch up.